SaitamaRobot/modules/stickers.py

```diff
@@ -376,15 +376,15 @@ def makepack_internal(
             success = context.bot.create_new_sticker_set(
                 user.id,
                 packname,
-                f"{name}s kang pack" + extra_version,
+                f"{name}'s kang pack" + extra_version,
```

Comment:
Gramatically makes more sense but as https://t.me/OnePunchDev/88178 said this would make Saitama ignore existing packs and create new packs, this causes high level of complains. 
So let's skip this one until we have a better solution for existing bots.

Reply:
Yea. Right 👍